docs/developer/architecture/core/uisettings-service.asciidoc

```diff
@@ -3,14 +3,68 @@
 
 NOTE: The UI settings service is available both server and client side.
 
-=== Server side usage
+=== Overview
+
+UI settings are configurable from the Advanced Settings page in Management and control the behavior of {kib}. `uiSettings` are stored in a `config` saved object and, as such, conform to the same conditions as other saved objects.
```

Comment:
nits/questions:
- Should we link to the saved objects docs from here?
- Do we capitalize Saved Objects in our docs? I'm never sure the right thing to do here, and I see a mix of both in the docs.

Reply:
Maybe it depends on the context? Here I'm referring to Saved Objects as entities (proper nouns?), so I guess we should capitalize here too. 🤷‍♀️ . Lets go with that and see.